puppet/manifests/default.pp

```diff
@@ -4,6 +4,29 @@
 
   class { 'docker':
     dns => $ipaddress_docker0,
+    docker_users => ['vagrant'],
+  }
+
+  # NOTE: This cert wrangling is only good for a single machine. We need some
+  # other mechanism to get our certs to the right place in a multi-node setup.
+  package { 'openssl': }
+  ->
+  openssl::certificate::x509 { 'docker-registry':
+    country => 'NT',
+    organization => 'seed-stack',
+    commonname => 'docker-registry.service.consul',
+    base_dir => '/var/docker-certs',
```

Comment:
Yes, that creates the cert we're using. The thing that changed in the most recent commit was how we tell the docker daemon about it.

(Using the system CA store is more flexible, and also requires less mucking about with hostnames in config dirs.)